app/classifier/components/TaskNavButtons/components/TalkLink/TalkLink.jsx

```diff
@@ -87,10 +87,17 @@ export function TalkLink({ disabled, onClick, projectSlug, subjectId, theme, tra
     );
   }
 
+  function onRightClick(e) {
+    setTimeout(() => {
+      onClick(e);
+    }, 100);
+  }
```

Comment:
Use a small delay here so that the link context menu opens for the old subject link, not the new subject link.

Reply:
I upped this to 1.5s, so that the transition to the next subject is less abrupt.